news/2024/2024-12-22-touhou-project-mania-cup-4-registrations-now-open.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: "Touhou Project Mania Cup 4: Registrations are Now Open!"
+date: 2024-12-22 12:00:00 +0000
+---
+
+![](/wiki/Tournaments/TMC/4th/image/banner@2x.png)
+
+With the celebration of the Touhou Project's 30th birthday, the Touhou Project Mania Cup is returning with its 4th iteration, bringing tons of improvement in competitiveness and player enjoyment!
+
+The [Touhou Project Mania Cup 4th](/community/forums/topics/2015815), or THMC 4 for short, is a team-based osu!mania 4k tournament. The tournament features 100% Touhou-related music and aims to bring top-tier competition to the tournament scene.
+
+We’ve had 80 teams [last year](/wiki/en/Tournaments/TMC/3rd), with **Bloxy cola** reaching the top with an undefeated performance. [This year](/wiki/en/Tournaments/TMC/4th), with more teams joining the competition, which team will rise to the challenge and become the best? 
+
+## What's new
+
+- **A Group Stage will debut this year** featuring a 16-group format. This means that we will have **64 teams** in the main event.
+  - If more than 70 teams play the qualifier, the group stage will use the split double elimination format.
+  - If fewer than 70 teams play the qualifier, the group stage will use the standard double elimination format.
+
+![Groups format](/wiki/Tournaments/TMC/4th/image/group-stage-bracket.png)
+
+  - **Each team will have 2 bans in the group stage**, providing extra strategic options throughout the different weeks.
+- **The "X" Picks will apply for RC and LN slots**, giving players more options to pick harder charts.
+- Two teams from each group will advance, with a total of 32 teams competing in a standard double-elimination bracket for the championship.
+
+## Custom charts 
+
+With almost **50 custom mappers** featured, this year we are bringing you a **nearly full custom** mappool from beginner to high-end. We hope this will not only enhance our Touhou journey but also contribute to the osu! community by adding more Touhou maps!
+
+You can check out [Supa7onyz](/beatmaps/artists/435)'s brilliant Grand Finals custom tiebreaker songs from last year as an example!
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="//player.bilibili.com/player.html?isOutside=true&aid=487714748&bvid=BV1sN411D7RV&cid=1176362169&p=1" frameborder="0" allowfullscreen></iframe>
```

Comment:
might need to ping @Walavouchey for this one, i don't know whether osu! supports embed from bilibili or not :(

Reply:
youtube is definitely preferred. i'd personally be ok with this bilibili embed, but only if you can rectify the following

- there's shouldn't be a weird watermark in the video (who is the uploader? are they related to the tournament?)
- the player should not autoplay
- sound should be enabled when you click play
- for the language part we'll need more opinions

the embed seems to have a bunch of unnecessary parameters. this would be a simpler version
```suggestion
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.bilibili.com/player.html?bvid=BV1sN411D7RV" frameborder="0" allowfullscreen></iframe>
```

Reply:
- the uploader of this video is the host ([`[GB]Sanae`](https://osu.ppy.sh/users/11238501)), through i also perfer to upload this videos on official youtube channel.
- the player is [`[GB]SuddenDeath`](https://osu.ppy.sh/users/6699923)
- It should be click to play
for last part i could do nothing for it.